domador.js

```diff
@@ -659,4 +659,12 @@ Domador.prototype.isVisible = function isVisible (el) {
   return visible;
 };
 
+Domador.prototype.openCodeFence = function openCodeFence (el) {
+  var fencinglanguage = this.options.fencinglanguage(el);
+  if (el.childNodes.length > 0 && el.childNodes[0].tagName === 'CODE') {
```

Comment:
You could do just `var child = el.childNodes[0]; if (el && el.tagName === 'CODE')`